pkg/fleets/controller.go

```diff
@@ -49,6 +49,7 @@ import (
 // Controller is a the GameServerSet controller
 type Controller struct {
 	logger              *logrus.Entry
+	threadiness         int
```

Comment:
So looking at other controllers, such as [Kubernetes controllers](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/job/job_controller.go#L137) or the [sample controller](https://github.com/kubernetes/sample-controller/blob/master/controller.go#L156) they have a `Run()` signature of `Run(threadiness int, stopCh <-chan struct{})` (although some of the naming differs from place to place).

Should we be aligning with the Kubernetes convention here / are we losing anything by not doing that. Maybe we lose a bit of ability for people to transfer Controller building knowledge when moving to Agones? (although it's minimal)

This isn't something I'm _super_ worried about, but figured it was worth raising - and if we find it a real issue down the line, it's pretty easy to back out of.

WDYT?

Reply:
That's a good discovery. I'll adjust to match it.